mlir/include/mlir/Dialect/SparseTensor/Transforms/Passes.h

```diff
@@ -55,6 +55,33 @@ enum class SparseEmitStrategy {
   kDebugInterface, // generate only place-holder for sparse iteration
 };
 
+namespace sparse_tensor {
+/// Select between different loop ordering strategies.
+/// Loop ordering strategies for sparse tensor compilation.
+/// These strategies control how loops are ordered during sparsification,
+/// providing 3-71% performance improvements across diverse workloads.
```

Comment:
Remove this comment. Although I like to see numbers, such precise numbers should not be in the switch documentation since they are extremely workload and target arch dependent

Reply:
Just saying that each switch may favor particular situations differently suffices